internal/datatransfer/monitor.go

```diff
@@ -17,17 +20,23 @@ type Monitor struct {
 	ticker                      *time.Ticker
 	lastSuccessfulSyncTimes     map[string]time.Time
 	lastSuccessfulSyncTimesLock sync.RWMutex
+	fsSync                      FileSync
 }
 
 func NewMonitor(workloadsManager *workload.WorkloadManager, configManager *configuration.Manager) *Monitor {
 	ticker := time.NewTicker(configManager.GetDataTransferInterval())
+
 	monitor := Monitor{
 		workloads:                   workloadsManager,
 		config:                      configManager,
 		ticker:                      ticker,
 		lastSuccessfulSyncTimes:     make(map[string]time.Time),
 		lastSuccessfulSyncTimesLock: sync.RWMutex{},
 	}
+	err := monitor.InitStorage()
+	if err != nil {
+		log.Error("Cannot init the storage")
```

Comment:
In this case user would get correct `Monitor` struct. Should we really logging error here if so why not to return it in case of issue with storage initialization or maybe update it to warn or trace?